src/bin/vip-slowlogs.ts

```diff
@@ -188,26 +191,30 @@ void command( {
 	envContext: true,
 	format: true,
 	module: 'slowlogs',
+	usage: baseUsage,
 } )
 	.option( 'limit', 'The maximum number of log lines', 500 )
```

Comment:
Sorry, just looking at this option description....
This uses the term "lines", but I think we should be consistent and use the term "entries" here as well. Unless of course they are not technically interchangeable?

Reply:
I don't believe either is more or less accurate. I personally prefer "log entries" because the logs include more than the SQL slow query; they include additional metadata for each entry. Using "log entries" instead of "log lines" also removes any confusion about whether a multiline slow SQL query would return as 1 entry or the number of log lines it generates.

Example with the additional metadata:

```
$ vip @example-app.production slowlogs --format=json
[
	{
		"timestamp": "2024-04-12T11:11:02Z",
		"rowsSent": "1",
		"rowsExamined": "1",
		"queryTime": "0.795678",
		"requestUri": "",
		"query": "SELECT ts FROM _vt.heartbeat WHERE keyspaceShard='exampleapp1234:-';"
	}
]
```

Reply:
I have the same logic for preferring "entries". I encourage you to make it so.